### PR TITLE
Flip transition

### PR DESCRIPTION
**Pull request by tbaranes**
Related to #155

Last transition from VCTransitionsLibrary Flip:
- Take a direction (only Left / Right are working right now, Top / Bottom are broken, but didn't find why yet)
- Default gesture is the Pan
- No custom parameters

I'm gonna take another look later for the Top / Bottom, but I think we can already merged this for the next release (since the main directions are working)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ All notable changes to this project will be documented in this file.
 - Add `NatGeoAnimator` to support NatGeo transition animation. It supports only a direction `NatGeo(direction)`, if no specified, the default values are `NatGeo(Left)`. [#155](https://github.com/JakeLin/IBAnimatable/issues/155)
 - Add `Turn` to support Turn transition animation. It supports only a direction `Turn(direction)`, if no specified, the default values are `Turn(Left)`. [#155](https://github.com/JakeLin/IBAnimatable/issues/155)
 - Add `CardsAnimator` to support Cards transition animation. It supports parameters `Cards(direction)`, if no specified, the default values are `Cards(Forward)`. [#155](https://github.com/JakeLin/IBAnimatable/issues/155)
+- Add `FlipAnimator` to support Flip transition animation. It supports parameters `Flip(direction)`, if no specified, the default values are `Cards(Left)`. [#155](https://github.com/JakeLin/IBAnimatable/issues/155)
```
Comment (JakeLin) [inline CHANGELOG.md]: there is a typo in `Cards(Left)`, should be `Flip(Left)` I think

**Comment by JakeLin** [comment]
👍  thank you very much, please merge it.

BTW: I am working on the Pinch gesture, there maybe a small bug in Portal animator when uses with gesture interaction, I am looking at it. Once it is done, will submit a PR and release 2.3. It will be a big release since all `VCTransitionsLibrary` are in `IBAnimatable` now. 👯

**Comment by tbaranes** [comment]
Awesome. Let me know if you need any helps before the next release, that one is a huge one 🎉 
I will merge this PR once travis has finished its build.

**Comment by JakeLin** [comment]
@tbaranes thanks, I am still working on the `PortalAnimator`, once it is done, we will have a big release 🎊

**Comment by tbaranes** [comment]
BTW, I just though that I didn't update the README for each new animator 😢 

Another though, shouldn't we divide the README in subREADME, with (or not) a summary in the main README? Right now, the main one is getting really big and harder to navigate in.

**Comment by JakeLin** [comment]
Yes, may move API section out of README.md thoughts? @tbaranes

**Pull request merged**